.github/workflows/sozisel.yml

```diff
@@ -0,0 +1,69 @@
+name: Sozisel CI
+
+on:
+  pull_request:
+    branches: [ master ]
+  push:
+    branches: [ master ]
+
+jobs:
+  backend:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Elixir
+      uses: erlef/setup-elixir@885971a72ed1f9240973bd92ab57af8c1aa68f24
```

Comment:
is there a reason why you use specific commit of this action?

Reply:
It was used in an example workflow for elixir here: https://github.com/actions/starter-workflows and I just used that example to create our workflow.